### PR TITLE
Jenkins and tests

### PR DESCRIPTION
**Pull request by kapitoshka438**

### CODE REVIEW

File: generateFastfile.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        use_automatic_signing: true,
+        team_id: "${TEAM_ID}"
+    )
+
+    version = get_version_number(
+      xcodeproj: "CubicExample.xcodeproj",
+      target: "CubicExample"
+    )
+
+    build = get_build_number(xcodeproj: "CubicExample.xcodeproj")
```
Comment (renaissanceGeek) [inline generateFastfile.sh:32]: We talked in our one-on-one about reading `version` and `build` from a separate file (like the `COBALT_VERSIONS` file in cubic-dev), which I think is a good idea.  Just for my information, where is this reading from now?  Is `version` the `CURRENT_PROJECT_VERSION` in project.pbxproj?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!groovy
+//Copyright (2018) Cobalt Speech and Language Inc.
```
Comment (renaissanceGeek) [inline Jenkinsfile:2]:
```suggestion
//Copyright (2020) Cobalt Speech and Language Inc.
```

File: Jenkinsfile [hunk 3]
```diff
@@ -16,7 +16,7 @@ def build() {
 		commit.setBuildStatus("build", "ERROR", "Build failed")
 		throw err
 	} finally {
-		deleteDir()
+		//deleteDir()
```
Comment (renaissanceGeek) [inline Jenkinsfile:19]: Is this intentionally commented out?  If so, why do we have the finally block at all?

Comment (kapitoshka438) [reply Jenkinsfile:19]: I'm trying to solve one problem. Xcode test don't became unsuccessful. Cubic became returning error when I try to recognize test audio file. Once I solve it, I will uncomment this.

File: generateFastfile.sh (hunk 1 above)
Comment (kapitoshka438) [reply generateFastfile.sh:32]: `cubic-dev`'s `COBALT_VERSIONS` contains versions of "external" libraries to import. Here we only need project's own version number and build number. Version number is located in the `Info.plist`, build number is in the `project.pbxproj`. No need to duplicate this values in a separate file. Xcode tools have convenient methods to read/write them.

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,38 @@
+#!groovy
+//Copyright (2020) Cobalt Speech and Language Inc.
+
+def build() {
+	try {
+		checkout scm
+
+		commit.setBuildStatus("build", "PENDING", "Building...")
+		sh "sh generateFastfile.sh"
+		sh "security -i unlock-keychain -p ${USER_PASSWORD} ~/Library/Keychains/login.keychain-db"
```
Comment (happyalu) [inline Jenkinsfile:10]: where is this USER_PASSWORD stored?

Comment (kapitoshka438) [reply Jenkinsfile:10]: ios-worker environment variables

Comment (jhollowayj) [reply Jenkinsfile:10]: I would like to see that documented in a read me file somewhere.

**Pull request merged**